src/rgw/rgw_rados.cc

```diff
@@ -4762,7 +4762,9 @@ int RGWRados::Object::complete_atomic_modification()
   }
 
   string tag = (state->tail_tag.length() > 0 ? state->tail_tag.to_str() : state->obj_tag.to_str());
+
   auto ret = store->gc->send_chain(chain, tag); // do it synchronously
+  ldout(store->ctx(), 0) << "ret= " << ret << dendl;
```

Comment:
this function, and the need to call delete_objs_inline() on failure, is why we do it synchronously. your changes are effectively throwing out that error code :(

Reply:
right;  a way to deal with this is desirable, but blocking the client unconditionally while it is performed is not a requirement